MediaAdminBundle/Controller/Admin/MediaController.php

```diff
@@ -39,6 +39,7 @@ public function cropAction(Request $request, $mediaId)
 
         if ($form->isValid()) {
             $data = $form->getData();
+            $saveMediaManager = $this->get('open_orchestra_media_admin.manager.save_media');
```

Comment:
the internal property `$saveMediaManager` could be suppressed, you could directly call the method from service in the controller